font-codegen/src/fields.rs

```diff
@@ -1117,6 +1126,9 @@ impl Field {
                 // noop
                 CustomCompile::Skip => return Default::default(),
             }
+        } else if let Some(attr) = self.attrs.compile_with.as_ref() {
+            let method = &attr.attr;
+            quote!( self.#method() )
```

Comment:
In the past we've been moving away from allowing arbitrary expressions in attributes, and although it might sometimes be more verbose, I believe anything we might want to express with this attribute *should* be expressible as a method (but not as a free function, since we do expect that this will often require access to the instance's internal state)